tests/integration_tests/functional/test_snapshot_editor.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests for snapshot-editor tool."""
+
+import platform
+
+import pytest
+
+import host_tools.cargo_build as host
+from framework import utils
+
+PLATFORM = platform.machine()
+MIDR_EL1 = hex(0x603000000013C000)
+
+
+@pytest.mark.skipif(
+    PLATFORM != "aarch64",
+    reason="This is aarch64 specific test.",
+)
+def test_remove_regs(uvm_nano, microvm_factory):
```

Comment:
Is there any value in testing with all guest kernels?

Reply:
isn't uvm_nano 5.10 only?

Reply:
Yeah, that's why I am asking if running in 4.14 could have different behavior.

Reply:
The test itself won't change behavior. Set of registers in the snapshot is same for all kernels.

Reply:
what about 6.1?

Reply:
Same

Reply:
my point here is that, we cannot be sure that it will always be the same, so maybe it makes sense to add tests for the rest of the kernels as well? WDYT @pb8o @ShadowCurse ?

Reply:
As I mentioned before, this test will behave same for any kernel, because MIDR_EL1 reg will be in the snapshot in any case. If we want to have a test, that verifies that the set of regs is the same on all kernels, it should be a separate test.

Reply:
You are right. That's fine by me then.